types/test.tsx

```diff
@@ -35,13 +36,18 @@ export async function testPureRender() {
 
   // helpers
   const {container, rerender, debug} = page
+  expectType<HTMLElement, typeof container>(container)
```

Comment:
This was `Element` before this change which caused https://github.com/testing-library/react-testing-library/pull/834#issuecomment-767796456